src/default-options.ts

```diff
@@ -15,5 +15,6 @@ export default {
   },
   beforeMiddleware: [],
   afterMiddleware: [],
-  beforeStart: []
+  beforeStart: [],
+  initialize: null
```

Comment:
@anastasios-workable This overlaps with beforeStart. Initialize runs after diamorphosis while beforeStart is another queued function that is run after middlewares are initialized. Wdyt?